src/commands/sources/soundcloud.js

```diff
@@ -0,0 +1,61 @@
+// SimpleMusic - Source
+const youtubedl = require('youtube-dl');
+const fetch = require('node-fetch');
+
+exports.alias = ['soundcloud', 'sc'];
+
+exports.url = /https{0,1}:\/\/w{0,3}\.*soundcloud\.com\/([A-Za-z0-9_-]+)\/([A-Za-z0-9_-]+)[^< ]*/i;
+
+exports.getStream = (url) => {
+    return youtubedl(url, ['-f bestaudio']);
+}
+
+exports.getInfo = (url) => {
+    return new Promise(async (resolve, reject) => {
+        youtubedl.getInfo(url, [], function(err, info) {
```

Comment:
https://user-images.githubusercontent.com/17349626/103175146-830dd600-48b3-11eb-9937-0f859d6a3d2f.mp4

There seems to be an issue with `youtubedl.getInfo()`

After some testing on this issue, **it does call the SoundCloud file!** So it appears to be an issue with this module. **I've uploaded a video displaying this...**

After requesting a SoundCloud track URL, it will instead search on YouTube for some similar titled video as the SoundCloud track.

I'm unsure with youtubedl on how to fix this, but maybe a SoundCloud API request is possible too...